config.yaml

```diff
@@ -0,0 +1,56 @@
+options:
+  ua_token:
+    default: ""
+    type: string
+    description: |
+      Ubuntu Advantage token you have received with your license of Anbox Cloud
+  ua_source:
+    default: ""
+    type: string
+    description: |
+      APT source to use instead of the automatic one generated from UA.
+      Overrides UA source if both are set.
+  ua_source_key:
+    default: ""
+    type: string
+    description: |
+      Key ID to import to the APT keyring in order to access the specified
+      source archive.
+      Required if ua_source is set.
+  ua_use_staging:
+    default: False
+    type: boolean
+    description: |
+      Use staging contracts API service
```

Comment:
Can this not be removed in favour of using the Pro subordinate that @mthaddon and co. maintain?

Reply:
We can definitely use it . But we have yet to make decision internally on how we will go forward with all these charms requiring pro, i.e either use the subordinate charm or just make it a documented pre-requisite for use and just block the charm at deployment. This will then give the user flexibility to configure pro using their own methods whichever they comfortable with.

Reply:
@jnsgruk Ideally yes. We though have to keep backward compatibility but also should try to rethink if we can improve. 

@jat-canonical I think we maybe drop these for now so we keep the initial version clean (we don't use them yet). We can bring them back only if we decide in our plan moving forward that they're mandatory for backward compatibility and fit with our plan to handle the Pro dependency going foward. Wdyt?

Reply:
@jnsgruk on that note, is the Pro subordinate how Juju envisions to handle Pro in the future or are there other plans for a more tight integration?